src/apollo-server/index.ts

```diff
@@ -86,9 +79,34 @@ export class OttofellerApolloServerProject extends TypeScriptAppProject {
 
     // ANCHOR Scripts
     this.package.addField('type', 'module')
-    const tasksToRemove = ['build', 'compile', 'package', 'post-compile', 'pre-compile', 'watch']
-    tasksToRemove.forEach(this.removeTask.bind(this))
-    this.addTask('build', {exec: 'node esbuild.config.js'})
+
+    this.removeTask('build')
```

Comment:
@quesabe let's add a very explicit comment on why we are doing this, in all templates.